docs/asyncapi.yaml

```diff
@@ -291,6 +348,82 @@ components:
       type: number
       description: Key field for calculation Data 
       example: 1234567
+    ApplicationId:
+      type: number
+      description: Key field for calculation Data 
```

Comment:
should this have the same description as the above? could it be rephrased to make it clearer what the difference is?